packages/medusa/src/loaders/models.ts

```diff
@@ -5,30 +5,76 @@ import { ClassConstructor, MedusaContainer } from "../types/global"
 import { EntitySchema } from "typeorm"
 import { asClass, asValue } from "awilix"
 
+function formatModelName(modelName) {
+  // Remove "Model" from the end of the string
+  let formattedName = modelName.replace('Model', '');
+
+  // Capitalize the first letter of the string
+  formattedName = formattedName.charAt(0).toUpperCase() + formattedName.slice(1);
```

Comment:
todo: you can use upperFirst from the utils package

Reply:
Are you sure its there? I don't see one. Only from lodash.

Reply:
sorry, I have detailed it [here](https://github.com/medusajs/medusa/pull/4398#discussion_r1241770832)